rakelib/plugins_docs_dependencies.rake

```diff
@@ -135,5 +139,15 @@ task :generate_plugins_version do
   require "pluginmanager/gemfile"
   require "bootstrap/environment"
 
+  module ::Bundler
```

Comment:
should we put that in `lib/bootstrap/bundler.rb` ?

Reply:
I'd rather contain this patch to when computing the plugin docs, I'm still not clear on why the current code is the way it is, and we only see this issue in this specific context.

Reply:
Then I suggest we document it to that effect - having monkey patches around without knowing exactly why they  were put in place makes it very difficult for someone else in the future to understand the intent of that patch and know if it is still valid and/or required.

Reply:
I may have exaggerated https://github.com/elastic/logstash/pull/10942/files#diff-6edbc43295e5dbad52e9758a080a2eb6R142

Reply:
perfect!!